### ISSUE TITLE
Crayon: Fontstyles

### ISSUE THREAD

**Issue by Protofall** [Protofall/Crayon-Game-Framework#130]
Similar to the draw array structs, I think it would be good to make font styles to prevent repeating the same parameters over again. This struct would contain

- The `crayon_font_mono/prop_t` struct/s
- `Poly_list_mode` (Might do something similar for the other draw structs since this param usually doesn't change
- `scale` //x and y
- `crayon_palette_t` struct
- `colour`
- `fade`
- `newline_height` (How much to jump by when newline is encountered)

And possibly more stuff. This will also mean the old font draw structs will be removed in its own issue once this is done.

This issue isn't urgent, but must be done before Crayon version 1.0.0
